### PR TITLE
Enable setting a preferred result on a query

### PR DESCRIPTION
**Pull request by dschmidt**
This is still a bit quirky in the recently played tracks, but doesn't seem to be worse than before

### CODE REVIEW

**Comment by dschmidt** [comment]
Can I haz review? cc @muesli @theli-ua

File: src/libtomahawk/playlist/TrackDetailView.cpp [hunk 1]
```diff
@@ -441,3 +441,11 @@ TrackDetailView::setBuyButtonVisible( bool visible )
 {
     m_buyButtonVisible = visible;
 }
+
+
+void
+TrackDetailView::onResultClicked( const Tomahawk::playlistinterface_ptr& playlist, const Tomahawk::result_ptr& result, const Tomahawk::query_ptr& fromQuery )
+{
+    fromQuery->setPreferredResult( result );
```
Comment (theli-ua) [inline src/libtomahawk/playlist/TrackDetailView.cpp]: Do you think it would be better to *just* move the result, ie not play item right away?

Comment (theli-ua) [inline src/libtomahawk/playlist/TrackDetailView.cpp]: Yeah, that would be ok, but with your change it - it'll start playing even if the other query was playing, right?

Comment (dschmidt) [inline src/libtomahawk/playlist/TrackDetailView.cpp]: Not sure

Comment (theli-ua) [inline src/libtomahawk/playlist/TrackDetailView.cpp]: yup, it does

**Comment by theli-ua** [comment]
Actually, at the very least this does not make things worse, and adds a feature. Thus I'll just merge this.

**Pull request merged**